techlibs/xilinx/synth_xilinx.cc

```diff
@@ -296,15 +304,21 @@ struct SynthXilinxPass : public ScriptPass
 				run("read_verilog -lib +/xilinx/xc6v_cells_xtra.v");
 			else if (family == "xc7")
 				run("read_verilog -lib +/xilinx/xc7_cells_xtra.v");
-			else if (family == "xcu" || family == "xcup")
+			else if (family == "xcu" || family == "xcup") {
+				// Also read xc7 blackboxes; as many old cells are upgraded
+				// automatically, and some designs rely on this
+				run("read_verilog -lib +/xilinx/xc7_cells_xtra.v");
 				run("read_verilog -lib +/xilinx/xcu_cells_xtra.v");
```

Comment:
This doesn't look good.  Which old cells do you care about here?  IMO we should either cherry-pick the cells into the xcu list (if there's just a handful of them), or just merge all the blackbox files into a single `cells_xtra.v` (if there are lots).

Reply:
Added the PLLE2 and MMCME2 cells for the purpose of this PR, as LiteX and possibly some other things need them (due to the high cross-compatibility), but it seems like this will only be temporary based on the comments below.